### DISCUSSION TITLE
Spacing for c::function:: directive

### DISCUSSION THREAD

**Discussion by rscohn2** [executablebooks/meta#286] [Q&A]
If I do this:
```
.. c:function:: dds_platform *dds_mpi(MPI_COMM comm, int sz)

.. py:function:: foo(a, b)
```
Then there is no space between 'int' and 'sz'. See below:
<img width="292" alt="sphinx_book_theme" src="https://user-images.githubusercontent.com/1936730/111701870-d2f49d00-8811-11eb-8fb3-9e680f3ff7d4.png">

If I use the default Alabaster theme, it looks ok. See below. Is there a configuration that will do better spacing?

<img width="492" alt="alabaster" src="https://user-images.githubusercontent.com/1936730/111701938-ec95e480-8811-11eb-8f74-e975c5d08e0f.png">

**Comment by choldgraf** [comment]
Hmmmm - I believe that we inherit this from the upstream theme (https://pydata-sphinx-theme.readthedocs.io/). Not sure if this has been reported there too, but it feels like the fix would just be some CSS to make this output styled nicely.

**Reply by chrisjsewell** [reply]
No its here: https://github.com/executablebooks/sphinx-book-theme/blob/bdcf1065f0fbf92f786082b78c86af7a280651df/src/scss/_page.scss#L232-L234

**Reply by chrisjsewell** [reply]
so indeed we should be able to fix that

**Reply by choldgraf** [reply]
gotcha - added: https://github.com/executablebooks/sphinx-book-theme/issues/307
